SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -38,304 +39,276 @@
+  Prandtl_Turb = 0.0;   /*!< \brief Turbulent Prandtl number. */
+
+  /*!
+   * \brief Auxilary types to store common aero coefficients (avoids repeating oneself so much).
+   */
+  struct AeroCoeffsArray {
+    su2double* CD = nullptr;      /*!< \brief Drag coefficient. */
+    su2double* CL = nullptr;      /*!< \brief Lift coefficient. */
+    su2double* CSF = nullptr;     /*!< \brief Sideforce coefficient. */
+    su2double* CEff = nullptr;    /*!< \brief Efficiency (Cl/Cd). */
+    su2double* CFx = nullptr;     /*!< \brief x Force coefficient. */
+    su2double* CFy = nullptr;     /*!< \brief y Force coefficient. */
+    su2double* CFz = nullptr;     /*!< \brief z Force coefficient. */
+    su2double* CMx = nullptr;     /*!< \brief x Moment coefficient. */
+    su2double* CMy = nullptr;     /*!< \brief y Moment coefficient. */
+    su2double* CMz = nullptr;     /*!< \brief z Moment coefficient. */
+    su2double* CoPx = nullptr;    /*!< \brief x Moment coefficient. */
+    su2double* CoPy = nullptr;    /*!< \brief y Moment coefficient. */
+    su2double* CoPz = nullptr;    /*!< \brief z Moment coefficient. */
+    su2double* CT = nullptr;      /*!< \brief Thrust coefficient. */
+    su2double* CQ = nullptr;      /*!< \brief Torque coefficient. */
+    su2double* CMerit = nullptr;  /*!< \brief Rotor Figure of Merit. */
+    int _size = 0;                /*!< \brief Array size. */
+
+    void allocate(int size);      /*!< \brief Allocates arrays. */
+
+    void setZero(int i);          /*!< \brief Sets all values to zero at a particular index. */
+    void setZero() {              /*!< \brief Sets all values to zero for all indices. */
+      for(int i=0; i<_size; ++i) setZero(i);
+    }
+
+    AeroCoeffsArray(int size = 0) : _size(size) { if(size) allocate(size); }
+
+    ~AeroCoeffsArray();
+  };
+  /*!
+   * \brief Scalar version of the coefficients type.
+   */
+  struct AeroCoeffs {
+    su2double CD,CL,CSF,CEff,CFx,CFy,CFz,CMx,CMy,CMz,CoPx,CoPy,CoPz,CT,CQ,CMerit;
+
+    void setZero() {
+      CD=CL=CSF=CEff=CFx=CFy=CFz=CMx=CMy=CMz=CoPx=CoPy=CoPz=CT=CQ=CMerit=0.0;
+    }
+
+    AeroCoeffs() { setZero(); }
+  };
+
+  AeroCoeffsArray InvCoeff;          /*!< \brief Inviscid pressure contributions for each boundary. */
+  AeroCoeffsArray SurfaceInvCoeff;   /*!< \brief Inviscid pressure contributions for each monitoring boundary. */
+  AeroCoeffs AllBoundInvCoeff;       /*!< \brief Total pressure contribution for all the boundaries. */
+
+  AeroCoeffsArray MntCoeff;          /*!< \brief Inviscid momentum contributions for each boundary. */
+  AeroCoeffsArray SurfaceMntCoeff;   /*!< \brief Inviscid momentum contributions for each monitoring boundary. */
+  AeroCoeffs AllBoundMntCoeff;       /*!< \brief Total momentum contribution for all the boundaries. */
+
+  AeroCoeffsArray SurfaceCoeff;      /*!< \brief Totals for each monitoring surface. */
+  AeroCoeffs TotalCoeff;             /*!< \brief Totals for all boundaries. */
```

Comment:
...
defining auxiliary types to store aerodynamic coefficients that were "repeated" for inviscid, momentum, viscous, total, etc. contributions. The auxiliary types manage allocation / destruction and common operations like setting everything to zero.